lib/gtfs/route_pattern.ex

```diff
@@ -37,4 +38,8 @@ defmodule Gtfs.RoutePattern do
       representative_trip_id: row["representative_trip_id"]
     }
   end
+
+  @spec in_route_id_set?(Csv.row(), MapSet.t(id())) :: boolean
```

Comment:
Should have type `MapSet.t(Route.Id())`
```suggestion
  @spec in_route_id_set?(Csv.row(), MapSet.t(Route.id())) :: boolean
```

Reply:
Good catch